README.md

```diff
@@ -134,7 +132,7 @@ repository-data/application/src/main/resources/hcm-config/configuration/modules/
 
 ### Development
 
-> For best UI developer experience run the local dev server in one Terminal window, and run the SASS or LESS compiler with Maven in another one (remember to 'cd' into the appropriate folder)!
+For best UI developer experience run the local dev server in one Terminal window, and run the frontend build process via `npm` in another one.
```

Comment:
Is this instead of `make frontend` or does via NPM have more advantages.  It's probably worth given both options, but saying what one does over the other.